tests/automation_framework/tests/work_order_tests/test_submit.py

```diff
@@ -1141,10 +1116,9 @@ def test_work_order_submit_requesterId_param_remove (self):
             self.test_obj.build_request_output['request_obj'])
 
         assert (
-                verify_test(
-                    result_response, 0,
-                    self.test_obj.build_request_output['pre_test_output'],
-                    self.test_obj.build_request_output['action_obj'])
+                check_negative_test_responses(
```

Comment:
Similar to above

Reply:
"Missing parameter requesterId" is the new error introduced by Ram. Hence the change is valid.